apps/ui/src/components/Onboarding.vue

```diff
@@ -21,15 +27,26 @@ const hasPendingTasks = computed(() =>
   Object.values(tasks.value).includes(true)
 );
 
+watch(
+  hasPendingTasks,
+  value => {
+    lsSet('onboarding', {
+      ...lsGet('onboarding'),
+      [web3.value.account]: !value ? false : undefined
+    });
+  },
+  { immediate: true }
+);
+
 onMounted(async () => {
-  if (web3.value.account) await usersStore.fetchUser(web3.value.account, true);
+  const pending = lsGet('onboarding')?.[web3.value.account] ?? true;
+  if (pending && web3.value.account)
```

Comment:
Did not seem to work, I tried putting console.log inside the if, and it seems to never be called.

I think on mount, `web3.value.account` is not ready yet